packages/cactus-plugin-satp-hermes/src/main/typescript/blo/transaction/transact-handler-service.ts

```diff
@@ -1 +1,101 @@
+import { Logger } from "@hyperledger/cactus-common";
```

Comment:
This looks great. Can you please add a few tests for this service? Namely at least one e2e transaction and one failure scenario, as a start. If for some reason this is not possible, let's see what functionality is missing.